### ISSUE TITLE
SoundRTS can not use screenreader api

### ISSUE THREAD

**Issue by kranid** [soundmud/soundrts#117]
## Description
When i try to run SoundRTS from source code using command python soundrts.py, i get message:
> Couldn't use ScreenReaderAPI.

 
## Steps to reproduce
1. clone repo;
2. Execute python soundrts.py

## Actual result
SoundRTS works, but NVDA does not announce anything and i get Message: 

> Couldn't use ScreenReaderAPI.
## environment
Windows 10+python 2.7+pygame 1.9.4.

**Comment by soundmud** [comment]
Thanks for the report. I couldn't reproduce the issue.

**Comment by soundmud** [comment]
Maybe you were using a 64 bits version of Python (ScreenReaderAPI.dll is 32 bits only).
Anyway, I have switched to accessible_output2, so you may not get this problem again.
